media/stylus/components/wiki/page-buttons.styl

```diff
@@ -34,6 +34,13 @@
     }
 }
 
+/* Users can style zone landing pages. We can't be sure that dark blue will look
+   good on background colors they choose. */
+body:not(.zone-landing) #edit-button {
```

Comment:
https://twitter.com/chriscoyier/status/414149535800127488 :smile:

If measurement shows that this noticeably affects performance, though, we should address it.